assets/countries/index.js

```diff
@@ -5,7 +5,8 @@ var countriesDomestic = [
   'Guernsey',
   'Jersey',
   'Isle of Man',
-  'United Kingdom'
+  'United Kingdom',
+  'UK'
```

Comment:
It falls out of sync with the rest of the countries so i'd rather not try and encourage abbreviations everywhere...